src/TailwindBuilder.php

```diff
@@ -28,6 +28,7 @@ class TailwindBuilder
     public function __construct(
         private readonly string $projectRootDir,
         string $inputPath,
+        private readonly string $configPath,
```

Comment:
It looks like a BC break. I think if we will add it to the end and give it a default value - we will be able to avoid BC breaks